sdk/js/src/User.ts

```diff
@@ -110,4 +117,26 @@ export class DVCPopulatedUser implements DevCycleUser {
 
         return new DVCPopulatedUser(user, options, this.getStaticData())
     }
+
+    toObject(): PropertiesOnly<DVCPopulatedUser> {
```

Comment:
probably dont need this anymore, this was attempting to serialize the DVCPopulatedUser instance so that it could be send to the client, but I think instead we should just exchange the `DevCycleUser` and create a populated user on either side of the boundary